{
    "expected": [
        {
            "@timestamp": "2023-11-02T16:18:57Z",
            "aws": {
                "lambda": {
                    "error": {
                        "location": "/var/task/lambda_function.py:lambda_handler:17",
                        "message": "exception",
                        "stack_trace": "  File \"/var/task/lambda_function.py\", line 15, in lambda_handler\n    raise Exception(\"exception\")\n",
                        "type": "Exception"
                    },
                    "message": "msg",
                    "request_id": "3f9d155c-0f09-46b7-bdf1-e91dab220855"
                }
            },
            "cloud": {
                "provider": "aws",
                "service": {
                    "name": "aws_lambda"
                }
            },
            "ecs": {
                "version": "8.11.0"
            },
            "event": {
                "original": "{\"timestamp\":\"2023-11-02T16:18:57Z\",\"level\":\"ERROR\",\"message\":\"msg\",\"logger\":\"root\",\"stackTrace\":[\"  File \\\"/var/task/lambda_function.py\\\", line 15, in lambda_handler\\n    raise Exception(\\\"exception\\\")\\n\"],\"errorType\":\"Exception\",\"errorMessage\":\"exception\",\"requestId\":\"3f9d155c-0f09-46b7-bdf1-e91dab220855\",\"location\":\"/var/task/lambda_function.py:lambda_handler:17\"}"
            },
            "log": {
                "level": "ERROR",
                "logger": "root"
            },
            "tags": [
                "preserve_original_event"
            ]
        },
        {
            "@timestamp": "2023-12-08T23:21:04.632Z",
            "aws": {
                "lambda": {
                    "error": {
                        "message": "some reference error",
                        "stack_trace": "ReferenceError: some reference error\\n    at Runtime.handler (file:///var/task/index.mjs:3:12)\\n    at Runtime.handleOnceNonStreaming (file:///var/runtime/index.mjs:1173:29)",
                        "type": "ReferenceError"
                    },
                    "message": {
                        "stackTrace": [
                            "ReferenceError: some reference error",
                            "    at Runtime.handler (file:///var/task/index.mjs:3:12)",
                            "    at Runtime.handleOnceNonStreaming (file:///var/runtime/index.mjs:1173:29)"
                        ]
                    },
                    "request_id": "405a4537-9226-4216-ac59-64381ec8654a"
                }
            },
            "cloud": {
                "provider": "aws",
                "service": {
                    "name": "aws_lambda"
                }
            },
            "ecs": {
                "version": "8.11.0"
            },
            "event": {
                "original": "{\"timestamp\":\"2023-12-08T23:21:04.632Z\",\"level\":\"INFO\",\"requestId\":\"405a4537-9226-4216-ac59-64381ec8654a\",\"message\":{\"errorType\":\"ReferenceError\",\"errorMessage\":\"some reference error\",\"stackTrace\":[\"ReferenceError: some reference error\",\"    at Runtime.handler (file:///var/task/index.mjs:3:12)\",\"    at Runtime.handleOnceNonStreaming (file:///var/runtime/index.mjs:1173:29)\"]}}"
            },
            "log": {
                "level": "INFO"
            },
            "tags": [
                "preserve_original_event"
            ]
        },
        {
            "@timestamp": "2023-12-08T23:21:04.646Z",
            "aws": {
                "lambda": {
                    "error": {
                        "message": "some reference error",
                        "stack_trace": "ReferenceError: some reference error\\n    at Runtime.handler (file:///var/task/index.mjs:3:12)\\n    at Runtime.handleOnceNonStreaming (file:///var/runtime/index.mjs:1173:29)",
                        "type": "ReferenceError"
                    },
                    "message": "errors logged -  ReferenceError: some reference error\n    at Runtime.handler (file:///var/task/index.mjs:3:12)\n    at Runtime.handleOnceNonStreaming (file:///var/runtime/index.mjs:1173:29) SyntaxError: some syntax error\n    at Runtime.handler (file:///var/task/index.mjs:4:12)\n    at Runtime.handleOnceNonStreaming (file:///var/runtime/index.mjs:1173:29)",
                    "request_id": "405a4537-9226-4216-ac59-64381ec8654a"
                }
            },
            "cloud": {
                "provider": "aws",
                "service": {
                    "name": "aws_lambda"
                }
            },
            "ecs": {
                "version": "8.11.0"
            },
            "event": {
                "original": "{\"timestamp\":\"2023-12-08T23:21:04.646Z\",\"level\":\"INFO\",\"requestId\":\"405a4537-9226-4216-ac59-64381ec8654a\",\"message\":\"errors logged -  ReferenceError: some reference error\\n    at Runtime.handler (file:///var/task/index.mjs:3:12)\\n    at Runtime.handleOnceNonStreaming (file:///var/runtime/index.mjs:1173:29) SyntaxError: some syntax error\\n    at Runtime.handler (file:///var/task/index.mjs:4:12)\\n    at Runtime.handleOnceNonStreaming (file:///var/runtime/index.mjs:1173:29)\",\"errorType\":\"ReferenceError\",\"errorMessage\":\"some reference error\",\"stackTrace\":[\"ReferenceError: some reference error\",\"    at Runtime.handler (file:///var/task/index.mjs:3:12)\",\"    at Runtime.handleOnceNonStreaming (file:///var/runtime/index.mjs:1173:29)\"]}"
            },
            "log": {
                "level": "INFO"
            },
            "tags": [
                "preserve_original_event"
            ]
        }
    ]
}
